develop/TOOLS/TOONuGet/NuGet_Naming_Conventions.md

```diff
@@ -0,0 +1,96 @@
+
+    - Another situation is the module that it has to do with. An example of this is the CI/CD packages:
+        - Skyline.DataMiner.CICD.DMApp.Common (common code for creating a package)
+        - Skyline.DataMiner.CICD.DMApp.Automation (specific code to convert an Automation solution to dmapp package)
+        - Skyline.DataMiner.CICD.DMApp.XXX (specific code to convert a XXX solution/repository to dmapp package)
+
+## Mandatory metadata
+
+```xml
+<PropertyGroup>
+    ...
+    <Authors>SkylineCommunications</Authors>
+    <Company>Skyline Communications</Company>
+    <Descrption>{Short description about the NuGet}</Description>
+    <PackageTags>Skyline;DataMiner</PackageTags>
+    <PackageProjectUrl>https://skyline.be</PackageProjectUrl>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    ...
+</PropertyGroup>
+```
+
+- Authors has to be without whitespace.
+- PackageTags need to have at least 'Skyline' and 'DataMiner'. Extra tags can be added in wanted.
+- Always include a readme file to explain how to use the NuGet.
+
+When creating NuGets via the [SLC SE Repository Manager](xref:TOOSLCSERepositoryManager), the pipeline will automatically add the default license and icon file. If a different license or icon needs to be used then this can be specified in the metadata.
+
+```xml
+<PropertyGroup>
+    ...
+    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+    <PackageIcon>Icon.png</PackageIcon>
+    ...
+</PropertyGroup>
+```
+
+## Restricted names
+
+- Skyline.DataMiner.Files.XXX (DataMiner DLLs - Autogenerated by tool from Data-Acquisition domain)
+- Skyline.DataMiner.Dev.XXX (DevPacks - Autogenerated by tool from Data-Acquisition domain)
+- Skyline.DataMiner.CICD.XXX (Managed by Data-Acquisition domain)
+
+## Names to be used
+
+- Skyline.DataMiner.Core.XXX
+    - Holds the items that have a direct link to DataMiner (AppPackage creation/installation, DataMinerSystem, ...)
+    - DxM will also fall under this. Take in account the have clear error handling in case the DxM is optional and not available.
+- Skyline.DataMiner.Utils.XXX
+    - Holds all the utilities like helpers, parsers, ...
+        - E.g.: SNMP trap parsing, rate calculations, table context menu, ...
+
+## Suggestions and reminders
+
+- Take in account the target framework. If you have a dependency to .NET Framework 4.6.2 (e.g.: some DataMiner DLLS) then you need to target that as well.
+    - In general .NET Standard 2.0 is a good choice as that gives the most flexibility to be used somewhere else.
+- Try to make use of the SDK project style. This is more future proof and a lot cleaner.
+    - .NET Framework can still be used, but that is easily done by changing the value to 'net462'.
+    ```xml
+    <PropertyGroup>
+        ...
+        <TargetFramework>netstandard2.0</TargetFramework> => <TargetFramework>net462</TargetFramework>
+        ...
+    </PropertyGroup>
+    ```
```

Comment:
Maybe add a part about adding a README.md file to the projects as well.
We could reference to the README.md of DataMinerSystem.Automation as an example perhaps.